.github/workflows/cypress-testing.yml

```diff
@@ -38,19 +38,18 @@ jobs:
             sleep 1
           done || exit 1
 
-      - name: Start Cloudflare Wrangler
-        run: npx wrangler pages dev static --port 8080 --binding USE_RELOADLY_SANDBOX=true RELOADLY_API_CLIENT_ID="$RELOADLY_SANDBOX_API_CLIENT_ID" RELOADLY_API_CLIENT_SECRET="$RELOADLY_SANDBOX_API_CLIENT_SECRET" &
-        env:
-          RELOADLY_SANDBOX_API_CLIENT_ID: ${{ secrets.RELOADLY_SANDBOX_API_CLIENT_ID }}
-          RELOADLY_SANDBOX_API_CLIENT_SECRET: ${{ secrets.RELOADLY_SANDBOX_API_CLIENT_SECRET }}
+      - name: Funding
+        run: yarn test:fund
 
       - name: Cypress run
         uses: cypress-io/github-action@v6
         with:
           build: yarn run build
-          start: yarn test:fund
+          start: npx wrangler pages dev static --port 8080 --binding USE_RELOADLY_SANDBOX=true RELOADLY_API_CLIENT_ID="$RELOADLY_SANDBOX_API_CLIENT_ID" RELOADLY_API_CLIENT_SECRET="$RELOADLY_SANDBOX_API_CLIENT_SECRET" &
```

Comment:
@EresDev you had this as it's own step before but that's not right as the env hadn't fully setup yet that's why I've made this change here.

Is this the correct way to pass reloadly env vars?

Reply:
https://github.com/ubq-testing/pay.ubq.fi/actions/runs/11504344113/job/32023622908#step:7:1

I just added the id and secret for the sandbox taken directly from the google doc and the errors are still present.

Reply:
I think there is logic missing inside the cards feature that is not throwing the error properly otherwise Cypress would fail, do you agree? 

The log style looks unfamiliar to me also so I'm not 100% where it's coming from, so I think it needs it's own task and shouldn't block this pr.